playground/nextjs-pages/.env.example

```diff
@@ -0,0 +1,7 @@
+# Force development mode
```

Comment:
No need to change anything in the playgrounds or examples for now unless it's really necessary.
I'd keep them as they are!

Reply:
gotcha